buildSrc/src/main/java/edu/wpi/gripgenerator/settings/DefinedParamType.java

```diff
@@ -41,7 +41,7 @@ public boolean isOutput() {
     }
 
     private final String type;
-    private DefinedParamState state;
+    private Optional<DefinedParamState> state;
```

Comment:
Why not just initialize it to `DefinedParamState.INPUT`, since that's what happens by default anyways?